model/security/distinguished_name.smithy

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+structure CreateDistinguishedNameParams{
+    nodes_dn: DistinguishedNamesDetails
+}
+
+list DistinguishedNamesDetails{
+    member: String
+}
+
+structure DistinguishedName{
+    nodes_dn: DistinguishedNamesDetails
+}
+
+list DistinguishedNamesList{
+    member: DistinguishedName
+}
```

Comment:
This can be simplified:

```suggestion
structure DistinguishedNames {
    nodes_dn: NodesDn
}

list NodesDn {
    member: String
}

map DistinguishedNamesMap {
    key: String,
    value: DistinguishedNames
}
```